src/field_theory/normal.lean

```diff
@@ -224,3 +233,49 @@ alg_equiv.ext (λ _, (algebra_map E K).injective
 /-- Restriction to an normal subfield as a group homomorphism -/
 def alg_equiv.restrict_normal_hom [normal F E] : (K ≃ₐ[F] K) →* (E ≃ₐ[F] E) :=
 monoid_hom.mk' (λ χ, χ.restrict_normal E) (λ ω χ, (χ.restrict_normal_trans ω E))
+
+end restrict
+
+section lift
+
+variables {F} {K} (E : Type*) [field E] [algebra F E] [algebra K E] [is_scalar_tower F K E]
+
+/-- Lift algebra homomorphism to a normal subfield -/
```

Comment:
Could you please write a docstring that is a bit more explicit? Please mention `F`, `E`, and `\phi`, etc... in the docstring.